samples/migrations/migrate_vectorstore_to_alloydb.md

```diff
@@ -335,6 +383,22 @@ The process of getting data from vector stores varies depending on the specific
     )
     ```
 
+4. (Optional) To add in large data from the iterator
+
+    ```python
+    iterator = get_all_pinecone_data(pinecone_index, batch_size=5)
+    documents_length = 0
+    for ids, embeddings, content, metadatas  in iterator:
```

Comment:
This can also be improved through batching the insertion like https://github.com/googleapis/langchain-google-alloydb-pg-python/blob/779f18a4d5e749e739ec6adb8e736c93e5d9d90d/src/langchain_google_alloydb_pg/utils/pgvector_migrator.py#L92